__tests__/pet.test.js

```diff
@@ -0,0 +1,139 @@
+const Pet = require("../src/pet");
+let pet = new Pet('Fido');
+beforeEach(() => {
+ pet = new Pet('Fido')
+})
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+  
+    it('sets the name property', () => {
+       expect(pet.name).toEqual('Fido');
+      });
+    
+    it('has a initial age of 0', () => {
+        expect(pet.age).toEqual(0);
+      });
+    
+    it('has a initial hunger of 0', () => {
+        expect(pet.hunger).toEqual(0);
+      });
+    
+    it('has a initial fitness of 10', () => {
+        expect(pet.fitness).toEqual(10);
+      });
+  });
+
+  describe('growUp', () => {
+    it('increments the age by 1', () => {
+      pet.growUp();
+        expect(pet.age).toEqual(1);
+    });
+   
+    it('increments the hunger by 5', () => {
+        pet.growUp();
+          expect(pet.hunger).toEqual(5);
+      });
+    
+     it('decrements the fitness by 3', () => {
+        pet.growUp();
+          expect(pet.fitness).toEqual(7);
+      });
+     it('throws an error if the pet is not alive', () => {
```

Comment:
This is a way of doing it. As an extra test example, you could check the other pet levels.